StatBot/Database/PostgresRepositories/PostgresUserRepository.cs

```diff
@@ -36,6 +36,14 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        protected virtual void Dispose(bool disposing)
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2953)